### PR TITLE
Adding history display

### LINKED ISSUE #12: Movie history
We already have all movies saved in the database with the corresponding cycle dates.
It should be feasible to display some sort of history (maybe combined with the search functionality).

### PR DESCRIPTION
**Pull request by CptPie**
This PR adds a notice for the last ended Cycle and the watched movies from this cycle.
This fixes #12.

image

This needed a new field "LastCycle" to be added to the data struct for the cycles.html serving.
The LastCycle field utilizes the GetPastCycles function to retrive the last ended cycle.
I also added a css class "cycleHistory" for future css styling (but none of my bussiness :P)

### CODE REVIEW

File: server.go [hunk 1]
```diff
@@ -473,6 +474,13 @@ func (s *Server) handlerRoot(w http.ResponseWriter, r *http.Request) {
 	data.Movies = common.SortMoviesByName(movieList)
 	data.VotingEnabled, _ = s.data.GetCfgBool("VotingEnabled", DefaultVotingEnabled)
 
+	cycles, _ := s.data.GetPastCycles(0, 1)
```
Comment (zorchenhimer) [inline server.go:477]: Don't ignore the error, please.

File: templates/cycle.html [hunk 2]
```diff
@@ -8,6 +8,15 @@
 {{ $votingEnabled := .VotingEnabled }}
 {{ $votesAvailable := .AvailableVotes }}
 
+{{if .LastCycle}}
+<div class="cycleHistory">
+	In the last Movienight on {{.LastCycle.EndedString}} we watched:
+	{{range .LastCycle.Watched}}
+	</br>* <a href="/movie/{{.Id}}">{{.Name}}</a>
```
Comment (zorchenhimer) [inline templates/cycle.html:15]: Use a `<ul>` here instead of manually making bullet points.

File: server.go (hunk 1 above)
Comment (CptPie) [reply server.go]: fixed in 
2784fe1

File: templates/cycle.html (hunk 2 above)
Comment (CptPie) [reply templates/cycle.html]: fixed in 2784fe1

**Pull request merged**